layers/+spacemacs/spacemacs-purpose/packages.el

```diff
@@ -83,11 +83,11 @@
       (add-hook 'purpose-mode-hook #'spacemacs/window-purpose-sync-popwin)
       (spacemacs/window-purpose-sync-popwin))))
 
-(defun spacemacs-purpose/init-purpose-popwin ()
+(defun spacemacs-purpose/init-spacemacs-purpose-popwin ()
   (use-package spacemacs-purpose-popwin
     ;; don't load spacemacs-purpose-popwin if popwin is excluded.
     ;; can't wrap `spacemacs-purpose/init-purpose-popwin' in a top-level `when'
```

Comment:
Not related to this PR, but while we're at it we can delete these 2 comments as well.